### DISCUSSION TITLE
Fail on malformed struct tags

### DISCUSSION THREAD

**Discussion by RPGillespie6** [swaggest/openapi-go#125] [Ideas]
Currently openapi-go just silently skips struct fields with malformed tags such as:
```
type req struct {
    Foo string `json: "foo"` // space after colon
}
```
or
```
type req struct {
    Foo string `json:"foo` // missing end quote
}
```

It would be nice to catch these issues at generation time rather than runtime.

**Comment by vearutop** [comment]
I think there is no API for struct tag validation in stdlib, but such issues are detected by `go vet` (which also runs as part of `go test`).

```
./t.go:4:2: struct field tag `json:"foo` not compatible with reflect.StructTag.Get: bad syntax for struct tag value
```

**Reply by RPGillespie6** [reply]
Ah, maybe I just need a linting step to detect this at compile time. Thanks for the suggestion!
